### PR TITLE
Dev kmr

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

**Comment by k-rister** [comment]
I would appreciate as many eyes as possible on this PR due to the fact that is has the potential to break existing installations.

File: bin/subprojects-install [hunk 1]
```diff
@@ -72,6 +72,35 @@ done
...
+    active=0
+    if [ -n "${subprojects[$name_link]}" ]; then
+        sp_type=`echo ${subprojects[$name_link]} | awk '{print $2}'`
+        case "$sp_type" in
+            "benchmark")
+                if [ "${type_dir}" == "benchmarks" ]; then
+                    active=1
+                fi
+                ;;
+            "tool")
```
Comment (rafaelfolco) [inline bin/subprojects-install:87]: wondering why "benchmark" and "tool" without 's' here and in the directory it is with 's'
```
subprojects/benchmarks/hwlatdetect
subprojects/benchmarks/cyclictest
subprojects/benchmarks/uperf
subprojects/benchmarks/tracer
subprojects/benchmarks/trafficgen
subprojects/benchmarks/oslat
subprojects/benchmarks/fio
subprojects/tools/ovs
subprojects/tools/tcpdump
subprojects/tools/ftrace
subprojects/tools/procstat
subprojects/tools/kernel
subprojects/tools/sysstat
subprojects/core/multiplex
subprojects/core/packrat
subprojects/core/workshop
subprojects/core/junk-drawer
subprojects/core/toolbox
subprojects/core/roadblock
subprojects/core/CommonDataModel
subprojects/core/rickshaw
```

**Review by rafaelfolco — APPROVED** [review]
lgtm, just make sure the missing 's' is ok

File: bin/subprojects-install (hunk 1 above)
Comment (k-rister) [reply bin/subprojects-install:87]: I agree that it does seem "odd", but its actually an inversion of the existing logic:

https://github.com/perftool-incubator/crucible/blob/master/bin/subprojects-install#L95-L105

Comment (k-rister) [reply bin/subprojects-install:87]: which I guess are influenced with the format that was chosen for this file originally:

https://github.com/perftool-incubator/crucible/blob/master/config/default_subprojects

**Pull request merged**